hack/clusters/create-vsphere.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Where is this script called? e2e tests?

Reply:
This is a straight copy-pasta from a series of scripts meant to be invoked locally/manually for dev testing.
Not used by anything else.